src/components/navigation-bar/navigation-bar.component.js

```diff
@@ -21,7 +26,10 @@ const NavigationBar = ({ navigationType = 'light', children, ariaLabel }) => {
 NavigationBar.propTypes = {
   children: PropTypes.node,
   ariaLabel: PropTypes.string,
-  navigationType: PropTypes.oneOf(['light', 'dark'])
+  /** color scheme of navigation component */
+  navigationType: PropTypes.oneOf(['light', 'dark']),
+  /** if 'true' the content will not be visible */
```

Comment:
```suggestion
  /** if 'true' the children will not be visible */
```

Reply:
Should it not be `children` instead of `content` as per Jamie's comment?